docs/scos/dev/back-end-development/data-manipulation/data-publishing/handling-data-with-publish-and-synchronization.md

```diff
@@ -664,13 +664,9 @@ class HelloWorldStorageDeleter implements HelloWorldStorageDeleterInterface
      */
     protected function store($idMessage, HelloWorldStorageTransfer $messageStorageTransfer): void
```

Comment:
Do we even need the 2nd param $messageStorageTransfer here?
I also think the docblock might need small adjustment.

Reply:
Maybe we can reduce that even more, like so:
```php
<?php

namespace Pyz\Zed\HelloWorldStorage\Business\Deleter;

// ..

class HelloWorldStorageDeleter implements HelloWorldStorageDeleterInterface
{
    /**
     * @param \Generated\Shared\Transfer\EventEntityTransfer[] $eventTransfers
     *
     * @return void
     */
    public function deleteCollectionByHelloWorldEvents(array $eventTransfers): void
    {
        $messages = SpyHelloWorldMessageQuery::create()
            ->filterByIdHelloWorldMessage_In($eventTransfers)
            ->find();

        foreach ($messages as $message) {
            $this->delete($message->getIdHelloWorldMessage());
        }
    }

    /**
     * @param int $idMessage
     *
     * @return void
     */
    protected function delete(int $idMessage): void
    {
        SpyHelloWorldMessageStorageQuery::create()
            ->filterByFkHelloWorldMessage($idMessage)
            ->delete();
    }
}
```